extraction/extractor.py

```diff
@@ -85,8 +86,14 @@ def __extract_data_from_html_body(self, node):
     def __validate_and_extract_content(self, node):
         """This function will extract valid content and return it"""
 
-        # This function will be updated in next iteration
-        _ = node
-        _ = self.soup
+        content_extractors = list()
+        content_extractors.append(quote_extractor.QuoteExtractor())
+        content_extractors.append(embedded_p_pins_extractor
+                                  .EPinterestPinExtractor())
```

Comment:
You can make this list global. Also, convert it into tuple.

Reply:
Updated accordingly! Thanks for the suggestions.